Watch errors when moving/deleting files/folders

```
OS: Win10, 
NPM: 5.3.0
Node: 8.4.0
```
```
events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: Error watching file for changes: EPERM
    at _errnoException (util.js:1041:11)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1359:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! docs-scaffold@0.1.0 dev: `gulp watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the docs-scaffold@0.1.0 dev script.
npm ERR! This is probably not a problem with npm. 
         There is likely additional logging output above.
```

Haven't googled it yet, but I can investigate more later if necessary.

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -38,6 +38,11 @@ const swaggerUi = [
     path.resolve(__dirname, "node_modules", "swagger-ui-docs-preset", "dist", "swagger-ui-docs-preset.css")
 ];
 
+function swallowError(error) {
+    console.error(error.toString());
+    this.emit("end");
+}
+
 gulp.task("docs", (cb) => {
     exec("npm run build:docs", function(err, stdout, stderr) {
         console.error(stderr);
@@ -56,7 +61,7 @@ gulp.task("docs:watch", () => {
     gulp.watch(
         [`${ docsFolder }/*.*`, `${ docsFolder }/**/*.*`],
         ["docs:dev", "connect:reload:docs"]
-    );
+    ).on("error", swallowError);
 });
 
 gulp.task("sass", () => {
@@ -87,7 +92,7 @@ gulp.task("sass:watch", () => {
     gulp.watch(
         [`${ styleFolder }/*.scss`, `${ styleFolder }/**/*.scss`],
         ["sass:dev", "connect:reload:sass"]
-    );
+    ).on("error", swallowError);
 });
 
 gulp.task("script", (cb) => {
@@ -118,7 +123,7 @@ gulp.task("script:watch", () => {
     gulp.watch(
         [`${ scriptFolder }/*.js`, `${ scriptFolder }/**/*.js`],
         ["script:dev", "connect:reload:script"]
-    );
+    ).on("error", swallowError);
 });
 
 gulp.task("vendor", (cb) => {
@@ -188,7 +193,7 @@ gulp.task("lib:watch", () => {
     gulp.watch(
         [`${ libFolder }/*.js`, `${ libFolder }/**/*.js`],
         ["docs:dev", "sass:dev", "script:dev", "connect:reload"]
-    );
+    ).on("error", swallowError);
 });
 
 gulp.task("clean", (cb) => {

